solar_system.rb

```diff
@@ -0,0 +1,52 @@
+# solar_system.rb
+
+require "terminal-table"
+
+class SolarSystem
+  attr_reader :star_name, :planets
+
+  def initialize(star_name, planets = [])
+    @star_name = star_name
+    @planets = []
+  end
+
+  def add_planet(planet)
+    @planets << planet
+  end
+
+  def list_planets
+    table = Terminal::Table.new do |t|
+      @planets.each_with_index do |planet, index|
+        t << ["#{index + 1}.  ", planet.name.capitalize]
+      end
+    end
+
+    return table
+  end
+
+  def new_planet
+    puts "What is the planet's name?"
+    name = gets.chomp
+    puts "What is the planet's color?"
```

Comment:
This code to interact with the user might fit better in `main.rb` than here in `SolarSystem`.